### PR TITLE
[4.0] Label for fieldset in admin login

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue https://github.com/joomla/backend-template/issues/112 .

### Summary of Changes
(Remove Tag Fieldset. In the login form, there is no need to group the input fields.)
Add label to fieldset.

### Testing Instructions
Code Review

### CODE REVIEW

**Comment by brianteeman** [comment]
Why is there no need for this to be a fieldset? 
A fieldset should be used when you need to show a relationship between multiple form inputs. 
Surely there is a relationship between username and password ?

**Comment by chmst** [comment]
It is a single form on a singele login screen. See also the linked issue and discussion 

The frontend login screen has no fieldset. There it would make more sense as it can occur often and everywhere.

**Comment by zwiastunsw** [comment]
I'm against it. There is no point in saying that something doesn't make sense. Sometimes we don't see it, but it doesn't mean it doesn't make sense. The information about the group of fields is important for the users of assistive technologies.
The screen reader informs the user that it is dealing with a group of fields and reads its legend.

**Comment by brianteeman** [comment]
Just because its a single form doesnt mean that it is not a fieldset

File: administrator/modules/mod_login/tmpl/default.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 ?>
 <form class="login-initial form-validate" action="<?php echo Route::_('index.php', true); ?>" method="post" id="form-login">
-	<fieldset>
+	<fieldset label="<?php echo Text::_('MOD_LOGIN_LOGIN_TITLE'); ?>">
```
Comment (wojsmol) [inline administrator/modules/mod_login/tmpl/default.php]:
```suggestion
	<fieldset label="<?php echo Text::_('MOD_LOGIN_LOGIN_TITLE'); ?>" class="sr-only">
```

**Comment by wilsonge** [comment]
Closing as a11y team lead is against

**Pull request closed without merging**